### ISSUE TITLE
Expansion

### ISSUE THREAD

**Issue by paul-harambee** [GuillemHerrera/StandardDistance#1]
Hi Guillem

Absolutely love that you've added this!

May I ask what you plan on doing next here?

I would like to help be a part of a dev effort to create a Spatial Statistics Toolbox for QGIS similar in nature to Esri's offering in which one finds Standard Distance (http://desktop.arcgis.com/en/arcmap/10.3/tools/spatial-statistics-toolbox/standard-distance.htm)

**Comment by GuillemHerrera** [comment]
Hi Paul, 

Thanks and I'll wish this Plugin help you and a lot of more people!

I developed this Plugin, in the first time, because I needed to run this type of analisys in Qgis ( I really love this project and her philosophy), and I hate to be forced to use ESRI software to acomplish my needs. Really, you can run a  Weighted Standard Distance Analisys with SAGA software, but in a version not supported with QGIS.

As you see, this is simple Qgis Plugin (compared for example with excelent Plugin QNEAT3),  and I developed it also like a personal PyQgis development exercise. 

Your idea for develop and Spatial Statistics Toolbox for QGIS is great, and a few of a ESRI algs are actually implemented in another Plugins or native algorithms (for Example I_Moran derivations). I think that maybe is better develop some interesting algorithm present in ESRI's software  but not in Qgis, for make the project stronger, and make the Gis users conversions to FOSS software easier. I would be happy to develop it jointly if my personal time allows me.

**Comment by paul-harambee** [comment]
I agree. I would be happy to help you on this journey if we could identify which tools we would like to make available in QGIS?

**Comment by GuillemHerrera** [comment]
Excellent! 

I'm create a new [repository ](https://github.com/GuillemHerrera/Qgis-Plugins-Development), for this achieve. 

We can discuse there, (and everyone that would to contribue) new plugins projects development.
